1-exercises/A-undefined/exercise.js

```diff
@@ -22,6 +24,7 @@ function sayHello() {
 let hello = sayHello();
 console.log(hello);
 
+// the function won't return anything, so the variable hello will return undefined.
```

Comment:
Nitpick: the variable hello will be undefined, try to avoid using the word `return` for variables and just use it for functions.